nano/node/bootstrap/bootstrap_ascending.cpp

```diff
@@ -182,40 +205,85 @@ void nano::bootstrap::bootstrap_ascending::account_sets::unblock (nano::account
 void nano::bootstrap::bootstrap_ascending::account_sets::force_unblock (const nano::account & account)
 {
 	blocking.erase (account);
-	backoff[account] = 0.0f;
-}
-
-std::vector<double> nano::bootstrap::bootstrap_ascending::account_sets::probability_transform (std::vector<decltype (backoff)::mapped_type> const & attempts) const
-{
-	std::vector<double> result;
-	result.reserve (attempts.size ());
-	for (auto i = attempts.begin (), n = attempts.end (); i != n; ++i)
-	{
-		result.push_back (1.0 / std::pow (2.0, *i));
-	}
-	return result;
 }
 
 nano::account nano::bootstrap::bootstrap_ascending::account_sets::random ()
```

Comment:
It would be nice to explain what this function does.
It is not easy to understand.

Reply:
Added clarity